packages/local_auth/local_auth_ios/ios/Classes/FLTLocalAuthPlugin.m

```diff
@@ -0,0 +1,229 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
After the webview federation, where I was concerned about the impact to `git blame` of the copy-and-delete approach, I came up with a different way of doing this. See
https://github.com/flutter/plugins/pull/4465
and
https://github.com/flutter/plugins/pull/4497
for a pair of PRs that allows for preserving history by temporarily making the app-facing plugin unpublishable and using path-based deps to allow CI to still validate everything.

You *should* (I haven't tried, but don't see why it wouldn't work) be able to retrofit the PR to do this without re-doing anything, by doing a new commit that deletes the app-facing versions of these files and then squashing it with the commit that copied them.

/cc @mvanbeusekom

Reply:
I can confirm that deleting the app-facing version and then squashing it with the previous commits works (see #4702).

Reply:
This is not showing as a move; it looks like you forgot to delete local_auth/ios/